src/runtime_src/CMakeLists.txt

```diff
@@ -40,7 +40,9 @@ if (XRT_DKMS_ALVEO STREQUAL "ON")
   xrt_add_subdirectory(ert)
 endif()
 
+# --- Optional HIP bindings ---
 if (XRT_ENABLE_HIP)
+  include(${CMAKE_CURRENT_SOURCE_DIR}/../CMake/hip.cmake)
```

Comment:
Can you move hip.cmake to src/runtime_src/hip/ some place and include it in src/runtime_src/hip/CMakeLists.txt?  We need to move away from global configuration.

Reply:
Done